src/regdesk/login/login_manager.js

```diff
@@ -0,0 +1,60 @@
+import { login } from '../../regfox/regfox_api.js';
+
+const STORAGE_LOCAL_KEY = 'regfox-bearer-details';
```

Comment:
Tempted to say these should live somewhere central. We can revisit it when it actually matters.